cognite/neat/workflows/model.py

```diff
@@ -156,3 +156,29 @@ def float_to_int(cls, value):
         if isinstance(value, float):
             return int(value)
         return value
+
+
+class WorkflowConfigs(BaseModel):
+    """Workflow configs"""
+
+    configs: list[WorkflowConfigItem] = []
```

Comment:
I think the list is the wrong data structure here, you spend a lot of time looking up items. Lets go for a dict.
```suggestion
    configs: dict[str, WorkflowConfigItem] = {}
```

Reply:
All changes for config management is for separate PR